src/distributors/base-installer.ts

```diff
@@ -85,7 +88,8 @@ export abstract class JavaBase {
   }
 
   // this function validates and parse java version to its normal semver notation
-  protected normalizeVersion(version: string): semver.Range {
+  protected normalizeVersion(version: string) {
+    let access = 'ga';
```

Comment:
Let's rename access to stable (true, false)
And i think we should resolve boolean stable to `ea` or `ga` later in getAvailableVersions method.

Reply:
Done. Thank you.